C++/Most_Useful_Algorithms/BSTiterator.cpp

```diff
@@ -0,0 +1,50 @@
+#include<bits/stdc++.h>
+using namespace std;
+
```

Comment:
Same drop the preprocessors and Add the link to the problem.

Reply:
added the problem link on top.